share/spice/just_delete_me/content.handlebars

```diff
@@ -3,13 +3,8 @@
     {{notes}}
   </p>
 {{/if}}
-{{#if delete_url}}
```

Comment:
Seeing as the intent of this Instant Answer is to have the developer follow through to a specific URL I think we need a very clear call to action. The title link is not very obvious (by design) because typically the instant answer provides all the info should't require you to click though. In this case the intent of the IA is to send them to a website and unless we make it clear, it's not immediately obvious.

I think we need to make this more clear by proving some context. For the instructions we need to indicate what exactly they represent, "Account deletion instructions".

Alongside the instructions we need to make the call to action more clear because they need to follow the URL to delete their account. We should provide a button that has some generic text that suggest this is the user's next step: "Delete my account", "Account Deletion Link", "Visit {{domain}}".

I just don't think it's very clear, especially in the "Plenty of Fish" case above -- what "form" are the instructions referring to? Where is the form? How do I get there?

Does that make sense?